pywbem_mock/_wbemconnection_mock.py

```diff
@@ -77,12 +77,17 @@
 # document our behavior in relation to the spec.
 
 
-def _display(dest, text):
+def _uprint(dest, text):
     """
     Display to dest defined by text. This function appends the data
     in text to the file defined by dest or to stdout of dest is None
     """
-    # TODO make this unicode file for python 2
+    # replace errors since there are issues with some windows versions.
+    if sys.version_info[0] == 2 and sys.version_info[1] == 6:
```

Comment:
You could use `sys.version_info[0:2] == (2, 6)` instead. Works even with `<` and `>`.